hadoop.yaml

```diff
@@ -0,0 +1,86 @@
+package:
+  name: hadoop
+  version: 3.4.0
+  epoch: 0
+  description: Open Source Identity and Access Management For Modern Applications and Services
+  copyright:
+    - license: Apache-2.0
+  target-architecture:
+    # note: skipping aarch64 because frontend-maven-plugin doesn't work on aarach arch.
+    - x86_64
+  dependencies:
+    runtime:
+      - openjdk-8
+
+environment:
+  contents:
+    packages:
+      - bash
+      - busybox
+      - ca-certificates-bundle
+      - maven
+      - nodejs
+      - openjdk-8
+      - wolfi-base
+      - wolfi-baselayout
+      - yarn
+  environment:
+    LANG: en_US.UTF-8
+    JAVA_HOME: /usr/lib/jvm/java-1.8-openjdk
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/apache/hadoop
+      tag: rel/release-${{package.version}}
+      expected-commit: bd8b77f398f626bb7791783192ee7a5dfaeec760
+
+  - uses: maven/pombump
+    with:
+      patch-file: pombump-deps.yaml
+      properties-file: pombump-properties.yaml
+      pom: hadoop-project/pom.xml
+
+  - runs: |
+      tar --version
+      mkdir -p ${{targets.contextdir}}/
+      mkdir -p ${{targets.contextdir}}/usr/share/java
+      mkdir -p ${{targets.contextdir}}/usr/bin
+      mvn -Pdist,src,yarn-ui -DskipTests -Dtar -Dmaven.javadoc.skip=true clean package -e -X
+      tar -xvf hadoop-dist/target/hadoop-${{package.version}}.tar.gz -C ${{targets.contextdir}}/usr/share/java
+      for f in ${{targets.contextdir}}/usr/share/java/hadoop-${{package.version}}/bin/*; do
+        ln -sf $f ${{targets.contextdir}}/usr/bin/$(basename $f)
```

Comment:
This is no-op. It's going to create links to files that don't exist as contextdir will be accounted for